app/src/debug/java/org/mozilla/focus/web/Config.kt

```diff
@@ -6,4 +6,5 @@ package org.mozilla.focus.web
 
 object Config {
     const val DEFAULT_NEW_RENDERER = false
+    const val EXPERIMENT_DESCRIPTOR_GECKOVIEW_ENGINE = "4eb60ebe-6bbc-4c26-8221-e446114a3813"
```

Comment:
Urk! This must have slipped when I reviewed the library. The descriptors should use the experiment name and not the ID. The ID is something Kinto generates and not really in our control.
I guess for this iteration we will need to ship as-is but I'll file an issue to get this fixed in a future release of Fretboard.

Reply:
Well, if we use the name instead, we need some way to specify the environment as well.